app/test/testutilsfunctions.cpp

```diff
@@ -0,0 +1,43 @@
+/***************************************************************************
+ *                                                                         *
+ *   This program is free software; you can redistribute it and/or modify  *
+ *   it under the terms of the GNU General Public License as published by  *
+ *   the Free Software Foundation; either version 2 of the License, or     *
+ *   (at your option) any later version.                                   *
+ *                                                                         *
+ ***************************************************************************/
+
+#include "testutilsfunctions.h"
+
+#include <QtTest/QtTest>
+#include <QtCore/QObject>
+
+const int DAY_IN_SECS = 60 * 60 * 24;
+const int MONTH_IN_SECS = 60 * 60 * 24 * 31;
+
+void TestUtilsFunctions::testFormatDuration()
+{
+  QDateTime t0 = QDateTime::currentDateTime();
+
+  testFormatDuration( t0, 0, QStringLiteral( "Invalid datetime" ) );
+  testFormatDuration( t0, 1, QStringLiteral( "0 minute ago" ) );
```

Comment:
How about "just now" instead of "0 minute ago" ?

Reply:
just pushed a fix to "a few seconds ago"

Reply:
updated to `just now`